src/Node/Process.purs

```diff
@@ -111,7 +111,7 @@ pid :: Pid
 pid = process.pid
 
 platform :: Platform
-platform = fromJust (Platform.fromString process.platform)
+platform = unsafePartial $ fromJust (Platform.fromString process.platform)
```

Comment:
No need to worry about this now, but just from a style perspective I don't like mixing `$` with parentheses in cases like this, I'd rather use `unsafePartial <<< fromJust $ Platform.fromString process.platform` or `unsafePartial (fromJust (Platform.fromString process.platform))`.